app/src/main/res/layout/fragment_home.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should we also remove XML layout setup here? Or you're planning to remove it in a new PR?